packages/react-button/src/components/ToggleButton/checkedBehaviorDefinition.ts

```diff
@@ -0,0 +1,16 @@
+import { BehaviorRule, Rule } from '@fluentui/a11y-testing';
```

Comment:
Let's remove this file, it's not relevant anymore

Reply:
removed :)